config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -120,6 +120,45 @@ periodics:
     testgrid-alert-email: skclark@redhat.com
     description: "Uses kubetest to run serial node-e2e tests (+Serial, -Flaky|Benchmark|Node*Feature)"
 
+- name: ci-kubernetes-node-arm64-ubuntu-serial
+  interval: 4h
+  cluster: k8s-infra-prow-build
+  annotations:
+    testgrid-dashboards: sig-node-kubelet
+    testgrid-tab-name: kubelet-gce-e2e-arm64-ubuntu-serial
+    description: "Run serial node e2e tests on ARM64 environment on Ubuntu"
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  spec:
+    containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230207-192d5afee3-master
+        args:
+          - --repo=k8s.io/kubernetes=master
+          - --timeout=240
+          - --root=/go/src
+          - --scenario=kubernetes_e2e
+          - --
+          - --deployment=node
+          - --env=KUBE_BUILD_FOR_ARM64=true
+          - --gcp-zone=us-central1-a
+          - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/arm/image-config.yaml
```

Comment:
This will likely need the new flags introduced in https://github.dev/kubernetes/kubernetes/pull/117017, i.e.

```
--use-dockerized-build=true --target-build-arch=linux/arm64
```

Reply:
Done, updated with the latest command.

Reply:
seems like there is a typo in the config file, 

```
--image-config-file=/workspace/test-infra/jobs/e2e_node/arm/image-config.yaml
```

while it should 
```
--image-config-file=/workspace/test-infra/jobs/e2e_node/arm/image-config-serial.yaml
```

Can you confirm?

/hold

Reply:
Good catch. Named as `image-config.yaml` in my own test. Updated to match.